recipes/joommf/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "joommf" %}
+{% set version = "0.8.3" %}
+{% set sha256 = "d1f31753d829d172b68a8720416f0e413c741b34b69235a8e324056db3264cf9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    
+  run:
+    - python >=3.5
+    - oommfc
+    - jupyter
+    - pytest
+
+test:
+  imports:
+    - joommf
+
+  commands:
+    - python -c "import sys; import joommf as m; sys.exit(m.test())"
+
+about:
+  home: http://joommf.github.io/
+  license: BSD 3-clause
+  summary: A Python interface to the OOMMF micromagnetic calculator
+  license_family: BSD
+  license_file: LICENSE
+
+  description: |
+    joommf is a meta package for JOOMMF project.
+
+  doc_url: http://joommf.readthedocs.io
+  dev_url: https://github.com/joommf/joommf
+
+extra:
+  recipe-maintainers:
+    - marijanbeg
+    - fangohr
+    - takluyver
+    - davidcortesortuno
+    - rpep
```

Comment:
@fangohr, @takluyver, @davidcortesortuno, @rpep, are you okay with being maintainers?

Reply:
Yes, fine with me

Reply:
Yes

Reply:
Yes, that's fine. Thank you!